pyvrp/Population.py

```diff
@@ -111,14 +116,20 @@ def select(self, rng: XorShift128) -> Tuple[Individual, Individual]:
         ----------
         rng
             Random number generator.
+        cost_evaluator
+            Cost evaluator to use when computing the fitness.
+        update_fitness
+            Boolean whether to update the fitness values before selecting the
+            parents. Can be set to False for efficiency if it is known that the
+            fitness has not changed, e.g. when calling select repeatedly.
```

Comment:
I'd add a kwargs here and say something like "any additional keyword arguments are passed to get_binary_tournament". That'd then also work well with #183 for `k`.

Reply:
Good idea but I've removed it completely like you suggested.